src/mockServiceWorker.js

```diff
@@ -21,8 +21,13 @@ self.addEventListener('activate', async function (event) {
 })
 
 self.addEventListener('message', async function (event) {
+  if (!event.source.id) return
   const clientId = event.source.id
+
+  if (!event.currentTarget.clients) return
```

Comment:
I believe we can use `this.clients.get` instead of this to ensure there is always a global scope to resolve this client retrieval operation. As it turns out, `event.target`/`event.currentTarget` can be `null` in Safari.